docs/index.html

```diff
@@ -224,6 +224,8 @@
       }
     </script>
     <script src="//cdn.jsdelivr.net/npm/vue@2/dist/vue.min.js"></script>
+    <!-- 回到顶部按钮 -->
+    <script src="../src/plugins/clickHeader.js"></script>
```

Comment:
Thank you for your PR. 

You can publish the plugin by yourself instead of submitting it to the repository.

It can be submitted to awesome. https://github.com/docsifyjs/awesome-docsify?tab=readme-ov-file#plugins